app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -200,7 +229,7 @@ You can set up a job alert to be updated about opportunities throughout the year
 
 ## Apply for your visa
 
-You can apply for a skilled worker visa once you have been offered a teaching job at a school which can sponsor them. Find out [how to apply for a skilled worker visa, what salary you will need to earn and other types of visa you may be eligible for](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration). 
+You can apply for a skilled worker visa once you have been offered a teaching job at a school which can sponsor them. Find out [how to apply for a skilled worker visa, what salary you will need to earn and other types of visa you may be eligible for](/non-uk-teachers/visas-for-non-UK-teachers). 
```

Comment:
is this link the main CTA for this section? should it maybe go at the end of the section, if they need to read the rest of the section first to decide whether they need to go here or not? It's also a bit of a long link - could it be snappier to just be something like 'Find out how to apply for a skilled worker visa'?